CHANGELOG.md

```diff
@@ -2784,7 +2784,11 @@ Other minor changes
   foldr-map : foldr f x (map g xs) ≡ foldr (g -⟨ f ∣) x xs
   foldl-map : foldl f x (map g xs) ≡ foldl (∣ f ⟩- g) x xs
   ```
-
+  
+* Added new function to `Data.List.Properties`
+  ```agda
+  take-all : {a : Level} {A : Set a} (k : ℕ) (l : List A) → (k ≥ length l) → take k l ≡ l
```

Comment:
Also for the CHANGELOG entries we typically only provide the "useful" part of the type. It's a little bit subjective, but generally variable declarations aren't necessary. So for example this one would become 
```
n ≥ length xs → take n xs ≡ xs`
```